app/javascript/src/Navigation/components/VerticalNav.jsx

```diff
@@ -39,9 +40,9 @@ const VerticalNav = ({ sections, activeSection, activeItem }: Props) => (
 const NavSection = ({title, isSectionActive, activeItem, items, outOfDateConfig}) => {
   return (
     <NavExpandable title={title} isActive={isSectionActive} isExpanded={isSectionActive} className={outOfDateConfig ? 'outdated-config' : ''}>
```

Comment:
Only a suggestion
```suggestion
  const shouldShowIcon = outOfDateConfig && !isSectionActive
  return (
    <NavExpandable title={title} isActive={isSectionActive} isExpanded={isSectionActive} className={shouldShowIcon ? 'outdated-config' : ''}>
```

Reply:
It doesn't work, I already tried `isSectionActive`, but it only changes it's value when you click and navigate to any of the subitems, not when you expand/collapse from other pages

Reply:
Right, it should be the state not the prop.